src/Library/demos/Banner/main.blp

```diff
@@ -0,0 +1,48 @@
+using Gtk 4.0;
+using Adw 1;
+
+Box {
```

Comment:
I think this containing box could be dropped, since everything else is in the status page.

Reply:
Can I drop that box? Because it contains two children, the banner and the overlay/status page.

Reply:
Oh, you're right, sorry.